pkg/cloudprovider/aws/apis/v1alpha1/provider_defaults.go

```diff
@@ -65,3 +69,36 @@ func (c *Constraints) defaultAMIFamily() {
 	}
 	c.AMIFamily = &AMIFamilyAL2
 }
+
+func (c *Constraints) defaultBlockDeviceMappings() {
+	if c.LaunchTemplate != nil {
+		return
+	}
+	if len(c.BlockDeviceMappings) != 0 {
+		return
+	}
+	defaultEBS := BlockDevice{
+		DeleteOnTermination: aws.Bool(true),
+		Encrypted:           aws.Bool(true),
+		VolumeType:          aws.String(ec2.VolumeTypeGp3),
```

Comment:
Do these have to be specified to the AWS API? Are there defaults we can rely on?